kerl

```diff
@@ -733,6 +743,63 @@ do_deploy()
     echo "kerl_deactivate"
 }
 
+is_valid_install_path()
+{
+    # don't allow installs into home directory
+    if [ "$1" = "$HOME" ]; then
+        echo "ERROR: You cannot install a build into $HOME. It's a really bad idea."
+        return 1
+    fi
+
+    # don't allow installs into .erlang because
+    # it's a special configuration file location
+    # for OTP
+    if [ "$1" = "$HOME/.erlang" ]; then
```

Comment:
The more I think about it, the more I think banning `.erlang` as a name is reasonable (with an appropriate error message indicating why, of course).